<?xml version="1.0"?>
<launch>

  <rosparam command="load" file="$(find platform_driver)/config/config.yaml" />
  <node pkg="platform_driver" type="platform_driver" name="platform_driver" output="screen">
  </node>
</launch>